src/main/kotlin/uk/gov/justice/digital/hmpps/assessrisksandneeds/api/model/OffenderAndOffencesDto.kt

```diff
@@ -0,0 +1,179 @@
+package uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class OffenderAndOffencesDto(
+  @Schema(description = "Offender CRN", example = "DX12340A")
+  val crn: String,
+
+  @Schema(description = "Gender", example = "MALE, FEMALE")
+  val gender: Gender,
+
+  @Schema(description = "date of birth", example = "1980-01-01")
+  @JsonFormat(pattern = "yyyy-MM-dd")
+  val dob: LocalDate,
+
+  @Schema(description = "Assessment date", example = "2021-01-01 19:24:45")
+  @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+  val assessmentDate: LocalDateTime,
+
+  @Schema(description = "Current Offence")
+  val currentOffence: CurrentOffence,
+
+  @Schema(description = "Date of first sanction for the offender", example = "2000-01-01")
+  @JsonFormat(pattern = "yyyy-MM-dd")
+  val dateOfFirstSanction: LocalDate,
+
+  @Schema(description = "Age at first conviction, conditional or absolute discharge in years", example = "19")
+  val ageAtFirstSanction: Int,
```

Comment:
I think this can be calculated from the DOB and dateOfFirstSanction so we should probably not make it mandatory unless dateOfFirstSanction is null. Or we should drop this and do the calculation ourselves?
I think both only exist because OASys RSR pre 6.26 asked for the age rather than the date